Ginger/Ginger/Variables/VariabelsListViewPage.xaml.cs

```diff
@@ -140,11 +140,18 @@ private void ShowHideEditPage(VariableBase variabelToEdit)
                 }
                 else if (mVariabelsParent is BusinessFlow)
                 {
-                    mVariabelEditPage = new VariableEditPage(mVarBeenEdit, mContext, showAsReadOnly, VariableEditPage.eEditMode.BusinessFlow);
+                    mVariabelEditPage = new VariableEditPage(mVarBeenEdit, mContext, showAsReadOnly, VariableEditPage.eEditMode.Default);
                 }
                 else if (mVariabelsParent is Activity)
                 {
-                    mVariabelEditPage = new VariableEditPage(mVarBeenEdit, mContext, showAsReadOnly, VariableEditPage.eEditMode.Activity);
+                    if(mPageViewMode== General.eRIPageViewMode.View)
```

Comment:
what is that? why you modified VariableEditPage view mode?

Reply:
i didnt do any change related to this, it might have came while merging from master (getting latest)